### PR TITLE
Documentation: .serviceworker implies .https these days

### PR DESCRIPTION
**Pull request by annevk**
See discussion at https://github.com/web-platform-tests/wpt/pull/18228.

### CODE REVIEW

File: docs/writing-tests/file-names.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ themselves precede any test type flag, but are otherwise unordered.
 
 
 `.https`
+`.serviceworker`
```
Comment (jgraham) [inline docs/writing-tests/file-names.md]: I think this kind of implies that `.serviceworker` is a thing that you can/should add by hand. I think maybe we shouldn't encourage that but should just document it in the documentation for `any` tests and note that it implies https.

Comment (annevk) [reply docs/writing-tests/file-names.md]: Ah, I guess the recommendation for normal service worker tests is to still use .https then?

Comment (jgraham) [reply docs/writing-tests/file-names.md]: Yes, `.serviceworker` is for the case where you want the automagically generated HTML wrapper for the `.js` file.

Comment (annevk) [reply docs/writing-tests/file-names.md]: Right, and only `any.js` so far.

**Comment by annevk** [comment]
Addressed feedback, still somewhat curious as to when this changed, but I guess it doesn't really matter.

**Review by jgraham — APPROVED** [review]

**Pull request merged**